packages/app/src/utils.tsx

```diff
@@ -431,3 +431,16 @@ export const formatNumber = (
     (options.unit ? ` ${options.unit}` : '')
   );
 };
+
+// format uptime as days, hours, minutes or seconds
+export const formatUptime = (seconds: number) => {
+  if (seconds < 60) {
+    return `${seconds}s`;
+  } else if (seconds < 60 * 60) {
+    return `${Math.floor(seconds / 60)}m`;
+  } else if (seconds < 60 * 60 * 24) {
+    return `${Math.floor(seconds / 60 / 60)}h`;
+  } else {
+    return `${Math.floor(seconds / 60 / 60 / 24)}d`;
```

Comment:
Would https://date-fns.org/v3.2.0/docs/formatDistance or `formatDistanceToNowStrictShort` (in the utils) not work here?

Reply:
hmm formatDistance could work, but don't like its verbose format. formatDistanceToNowStrictShort worked, but I ran into weird bug with seconds being added between re-renders